src/Badges.sol

```diff
@@ -330,13 +370,9 @@ contract Badges is
     return symbol_;
   }
 
-  function tokenURI(uint256 _tokenId)
-    external
-    view
-    virtual
-    override
-    returns (string memory)
-  {
+  function tokenURI(
```

Comment:
Why did this linting stuff change? did you override some local config by any chance?